plugins/plugins64.xml

```diff
@@ -81,6 +81,24 @@
 			</x64>
 		</install>
 	</plugin>
+	<plugin name="Code::Stats">
+		<x64Version>1.0.1</x64Version>
+		<aliases>
+			<alias name="CodeStats"/>
+		</aliases>
+		<description>Code::Stats - Write code, level up, show off! A free stats tracking service for programmers. This plugin enables XP tracking in Notepad++.</description>
+		<author>p0358</author>
+		<homepage>https://github.com/p0358/notepadpp-CodeStats</homepage>
+		<sourceUrl>https://github.com/p0358/notepadpp-CodeStats</sourceUrl>
+		<latestUpdate>2017-02-30\n- Added the ability to change API URL in GUI\n- Added user-agent for API pulses\n- Some bug fixes</latestUpdate>
+		<stability>Good</stability>
+		<install>
+			<x64>
+				<download>https://github.com/p0358/notepadpp-CodeStats/releases/download/v1.0.1/notepadpp-CodeStats_x64.zip</download>
+				<copy from="CodeStats.dll" to="$PLUGINDIR$\" validate="true"/>
```

Comment:
MD5 of CodeStats.dll needs to be added to validate.json
e.g.

```
    "JavaScript Map Parser":
    {
        "5f248d59e768a4702ab73f7cb6dc77e6": "JsMapParser.NppPlugin.dll"
    }
```,

Reply:
Added